projects/lights_control/src/lights_gpio.c

```diff
@@ -0,0 +1,212 @@
+#include "lights_gpio.h"
+#include "event_queue.h"
+#include "gpio.h"
+#include "gpio_mcu.h"
+#include "lights_events.h"
+#include "status.h"
+
+#define LIGHTS_HORN_PERIPHERAL 0
```

Comment:
What are these supposed to represent? Why are they defined here?

Reply:
this changed in the next review.